docs/readmes/orc8r/debug_logs.md

```diff
@@ -28,7 +28,7 @@ kubectl --namespace orc8r exec -it ${ORC_POD} -- /var/opt/magma/bin/service303_c
 
 ## Tail logs
 
-Prerequisite: install [`stern`](https://github.com/wercker/stern)
+Prerequisite: install [`stern`](https://github.com/stern/stern)
```

Comment:
Here, and in the other versions, I am actually not sure if that makes sense: `Fork of discontinued [wercker/stern](https://github.com/wercker/stern)`, are we sure this is still working and this code should be used if the original repo was discontinued?

Reply:
Otherwise we would have to throw everything out because this is a prerequisite. The fork has 1.2k stars and is being actively developed.